CHANGELOG.md

```diff
@@ -17,7 +17,9 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
-## [Unreleased]
+## [7.74.4] - 2025-04-04
```

Comment:
Please keep simulator related entries as «unreleased» (although they technically are released). 

Also, we don’t usually make a new release when there’s just type annotation related changes.

Reply:
Reverted version bump